admin_guide/managing_networking.adoc

```diff
@@ -320,6 +320,14 @@ the number of local MAC addresses that the network interface can handle before
 falling back to filtering MAC addresses in software.
 ====
 
+[IMPORTANT]
+====
+Currently, the egress router is not compatible with Amazon AWS due to AWS not
+compatible with macvlan traffic.
+
+See link:https://aws.amazon.com/answers/networking/controlling-vpc-egress-traffic/[the Amazon AWS documentation for alternate methods for egress routing].
```

Comment:
No, there's nothing you can suggest to users here. Nothing that AWS does is going to help, because OpenShift itself doesn't take advantage of those mechanisms, and there's no way they can take advantage of them from within pods.